src/main/kotlin/app/revanced/patches/shared/settings/preference/impl/SwitchPreference.kt

```diff
@@ -16,15 +13,21 @@ import org.w3c.dom.Element
  * @param default The default value of the switch.
  * @param summaryOn The summary to show when the preference is enabled.
  * @param summaryOff The summary to show when the preference is disabled.
+ * @param userDialogMessage dialog message to show, if the user changes setting from the default value.
  */
 internal class SwitchPreference(
     key: String, title: StringResource,
     val default: Boolean = false,
     val summaryOn: StringResource? = null,
-    val summaryOff: StringResource? = null
+    val summaryOff: StringResource? = null,
+    userDialogMessage: StringResource? = null
```

Comment:
Is this appropriate here? Aren't their other preferences that might need a `userDialogMessage`? If there might, you can create an abstract "`BaseDialogMessagePreference`" class.